### PR TITLE
SDI-903: ⬆ Latest spring boot for netty vuln

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 orbs:
-  hmpps: ministryofjustice/hmpps@3.14
+  hmpps: ministryofjustice/hmpps@7
 
 jobs:
   validate:
-    executor: hmpps/java
+    executor:
+      name: hmpps/java
+      tag: "17.0"
```
Comment (petergphillips) [inline .circleci/config.yml:10]: Have had to move to java 17 as jackson 2.15.2 only supports java 17

File: build.gradle.kts [hunk 2]
```diff
@@ -30,26 +30,23 @@ fun isNonStable(version: String): Boolean {
 }
 
 group = "uk.gov.justice.hmpps.gradle"
-version = "4.8.8"
+version = "4.9.0"
```
Comment (petergphillips) [inline build.gradle.kts:33]: can't go to `5` here as already used by the spring boot 3 branch

File: src/test/kotlin/uk/gov/justice/digital/hmpps/gradle/configmanagers/AppInsightsConfigManagerTest.kt [hunk 3]
```diff
@@ -1,41 +1,26 @@
 package uk.gov.justice.digital.hmpps.gradle.configmanagers
 
 import org.assertj.core.api.Assertions.assertThat
-import org.assertj.core.api.Assertions.tuple
-import org.assertj.core.groups.Tuple
-import org.gradle.api.tasks.Copy
-import org.junit.jupiter.api.Test
-import uk.gov.justice.digital.hmpps.gradle.UnitTest
+import org.gradle.testkit.runner.TaskOutcome
```
Comment (petergphillips) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/gradle/configmanagers/AppInsightsConfigManagerTest.kt:4]: tests copied over from spring boot 3 branch

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
```
Comment (mikehalmamoj) [inline gradle/wrapper/gradle-wrapper.properties:3]: I think the Gradle 8 upgrade won't be too hard after this PR. Maybe oneto consider later.

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**